submitted/rainbond-operator/README.md

```diff
@@ -0,0 +1,74 @@
+| `rainbondOperator.image.tag`        | rainbond-operator container image tag         | `v1.0.0-beta2`                                                                                                                                                                 |
+| `rainbondOperator.image.pullPolicy` | rainbond-operator container image pull policy | `IfNotPresent`                                                                                                                                                           |
+| `openapi.name`                      | openapi name                                  | `openapi`                                                                                                                                                                |
+| `openapi.image.repository`          | openapi container image                       | `registry.cn-hangzhou.aliyuncs.com/goodrain/rbd-op-ui`                                                                                                                                                      |
+| `openapi.image.tag`                 | openapi container image tag                   | `v1.0.0-beta2`                                                                                                                                                                 |
+| `openapi.image.pullPolicy`          | openapi container image pull policy           | `IfNotPresent`                                                                                                                                                           |
+| `openapi.image.port`                | openapi service port                          | `8080`                                                                                                                                                                   |
+| `openapi.image.nodePort`            | openapi service nodePort                      | `30008`                                                                                                                                                                  |
+| `openapi.image.downloadURL`         | rainbond package download url                 | `Official download address` |
+| `openapi.image.downloadMD5`         | rainbond package SHA256                 | `Official package SHA256` |
+| `openapi.image.installMode`         | The installation mode, WithPackage or WithoutPackage     | `WithoutPackage` |
+| `openapi.image.imageRepository`     | The default image address              | `Oregistry.cn-hangzhou.aliyuncs.com/goodrain` |
+
+
+Specify each parameter using the `--set key=value[,key=value]` argument to `helm install`. For example:
+
+```bash
+$ helm install --name rainbond-operator --set image.tag=v1.0.0-beta2 ./chart
```

Comment:
`helm install --name` is helm 2.0 syntax. 

3.0 syntax looks like:
```
helm install [NAME] [CHART] [flags]
```

Reply:
changed